src/lib/Util.ts

```diff
@@ -12,9 +12,11 @@ type CmdResult = {
 };
 
 export class Util {
-    public path: string = '';
+    public path: string = "";
+    public flowPath: string = "";
     constructor(public workspaceRoot: string, private logger: Logger) {
-        this.path = vscode.workspace.getConfiguration('git').get('path') || "";
+        this.path = vscode.workspace.getConfiguration("git").get("path") || "";
+        this.flowPath = vscode.workspace.getConfiguration("gitflow").get("path") || `"${this.path}" flow`;
```

Comment:
I think it is not necessary to introduce new class property. Either `this.path` shuod become just `path` or something like this

```
this.path = vscode.workspace.getConfiguration("gitflow").get("path") || `"${this.path}" flow`;
```

And then continue to use it.